src/executioners/OptimizeSolve.C

```diff
@@ -289,12 +318,12 @@ OptimizeSolve::objectiveFunction()
 {
   _form_function->updateParameters(*_parameters.get());
 
-  _problem.execute(EXEC_FORWARD);
+//  _problem.execute(EXEC_FORWARD);
```

Comment:
@lynnmunday @zachmprince  Do we need this? The tests pass without this. If this is needed at times, this may need to be included in the applyHessian routine.